<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeWidget</class>
 <widget class="QWidget" name="WelcomeWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>480</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Chatbot</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#initStackWidget {
background-image: url(:/bg/welcome.png);
}

#welcomePage {
background-color: rgba(255, 255, 255, 0);
}

* {
  color: rgb(79, 79, 79);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="initStackWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="welcomePage">
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="welcomePageLayout">
         <item row="6" column="0" colspan="5">
          <spacer name="spacerBottom">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1">
          <widget class="QToolButton" name="createChatbotButton">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Create new</string>
           </property>
           <property name="icon">
            <iconset resource="../res/chatbot_theme.qrc">
             <normaloff>:/icons/new_chatbot.png</normaloff>:/icons/new_chatbot.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QToolButton" name="openChatbotButton">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Open existent</string>
           </property>
           <property name="icon">
            <iconset resource="../res/chatbot_theme.qrc">
             <normaloff>:/icons/open_chatbot.png</normaloff>:/icons/open_chatbot.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="5">
          <spacer name="spacerTop">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>26</width>
             <height>202</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="2">
          <widget class="QToolButton" name="openLastChatbotButton">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Open last</string>
           </property>
           <property name="icon">
            <iconset resource="../res/chatbot_theme.qrc">
             <normaloff>:/icons/open_last_chatbot.png</normaloff>:/icons/open_last_chatbot.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <spacer name="spacerRight">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <spacer name="spacerLeft">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0" colspan="5">
          <spacer name="spacerSep">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QLabel" name="welcomeLabel">
           <property name="text">
            <string>Welcome! What do you want to do?</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="5">
          <widget class="QLabel" name="linkLabel">
           <property name="text">
            <string notr="true">&lt;a href=&quot;http://www.daleaceptar.gob.ar/&quot;&gt;www.daleaceptar.gob.ar&lt;/a&gt;&lt;/p&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="5">
          <spacer name="linkSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>64</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/chatbot_theme.qrc"/>
 </resources>
 <connections/>
</ui>
